app/TriggerEmail/TriggerEmail.php

```diff
@@ -306,6 +306,28 @@ public function spawnEmail(array $input): void
                     '[[env(REGISTRY_IMAGE_URL)]]' => config('speedi.system.registry_image_url'),
                 ];
 
+                break;
+            case 'AFFILIATION_VERIFIED': 
+                $template = EmailTemplate::where('identifier', $identifier)->first();
+                $affiliation = Affiliation::where([
+                    'id' => $to,
+                    'current_employer' => true
+                ])->first();
+
+                $newRecipients = [
+                    'id' => $to,
+                    'email' => $affiliation->email,
+                ];
+
+                $validationHours = round((int)config('speedi.system.otp_affiliation_validity_minutes')/60,0);
+                $replacements = [
+                    '[[env(SUPPORT_EMAIL)]]' => config('speedi.system.support_email'),
+                    '[[env(APP_NAME)]]' => config('speedi.system.app_name'),
+                    '[[env(REGISTRY_IMAGE_URL)]]' => config('speedi.system.registry_image_url'),
+                    '[[AFFILIATION_VERIFICATION_PATH]]' => config('speedi.system.portal_url') . '/user/profile/affiliations?verify=' . $affiliation->verification_code,
+                    '[[env(OTP_AFFILIATION_VALIDITY_HOURS)]]' => $validationHours,
```

Comment:
Did this come from Wendi, as early talks with Emily, the idea around limiting verification times wasn't received well and later removed everywhere.